packages/features/ee/billing/api/webhook/_invoice.paid.org.ts

```diff
@@ -36,6 +38,41 @@ async function handlePaymentReceivedForOnboarding({
   });
 }
 
+async function increaseRatelimitForOrganizationOwner(orgOwnerEmail: string) {
+  const user = await prisma.user.findUnique({
+    where: { email: orgOwnerEmail },
+    select: {
+      id: true,
+    },
+  });
+  if (!user) {
+    logger.error(`User not found for email: ${orgOwnerEmail}`);
+    return;
+  }
+
+  const { UNKEY_ROOT_KEY } = process.env;
```

Comment:
this check is cheaper than doing the db lookup, so maybe you want to run it before
it probably doesn't matter much though